lib/dynflow/web/console.rb

```diff
@@ -49,6 +50,21 @@ class Console < Sinatra::Base
         erb :worlds
       end
 
+      get('/export') do
+        set_download_headers('index.tar.gz')
+        Exporters::Tar.full_html_export(find_execution_plans(true, false), self)
```

Comment:
I'm not sure we should set `show_all` to true and `paginate` to false. This could pretty much get down the whole instance with rich execution plans history.